quickstarts/aws/amazon-athena/config.yml

```diff
@@ -15,14 +15,15 @@ description: |-
 
   ### More info
 
-  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-athena-monitoring-integration) to learn more about New Relic monitoring for Amazon Athena. 
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-athena-monitoring-integration) to learn more about New Relic monitoring for Amazon Athena.
```

Comment:
@bradleycamacho any suggestions on this description? Similar to my other comment in another PR, why not take the content from the docs page and use that?

New Relic offers an integration for reporting your Amazon Athena data. This document explains how to activate this integration and describes the data that can be reported.